docs/features/deserializing-events.md

```diff
@@ -30,4 +34,43 @@ When using official Azure events, you can use `.ParseFromData<>` to deserialize
 EventGridBatch<EventGridEvent<StorageBlobCreatedEventData>> eventGridBatch = EventGridParser.ParseFromData<StorageBlobCreatedEventData>(rawEvent);
 ```
 
+### Deserializing Both Cloud and EventGrid Events
```

Comment:
Now it's not really clear what the section is about, how about this?
```suggestion
### Deserializing Events with CloudEvent or EventGrid schema
```

Reply:
I would actually create 2 sections, one per schema.

Reply:
Done.